### ISSUE TITLE
Ignoring same files locally as when run as an action

### ISSUE THREAD

**Issue by bazzadp** [github/super-linter#985]
When I run this locally (on a Mac) it includes files/folders (like `node_modules`) that are ignored when run through GitHub Actions

What I'm running:

```
docker run -e RUN_LOCAL=true \                                                                                                                   
         -e VALIDATE_ALL_CODEBASE=false \
         -e VALIDATE_HTML=true \
         -v $PWD:/tmp/lint github/super-linter
```

I saw #179 but that seems to be about dynamically ignoring folders - I just want to ignore the files and folders that the super-linter knows to ignore anyway when run as aa GitHub Action.

Am I doing something wrong or is this not supported when it runs locally?

**Comment by GaboFDC** [comment]
These files should be ignored locally too. Do you mind running with the debug flags, and sharing the output?
An example of how to run is on the issue template.

**Comment by bazzadp** [comment]
Well that's interesting!

If I add `-e ACTIONS_RUNNER_DEBUG=true` it works as expected (though outputs a LOT of debugging info, that I'd rather not see unless I'm actually debugging the linter!).

However if I don't include that (or set it to `false`) it spends ages gathering the files:

```
2020-11-11 18:30:56 [INFO]   Gathering user validation information...
2020-11-11 18:31:25 [INFO]   ---------------------------------
2020-11-11 18:31:25 [INFO]   ------ File list to check: ------
2020-11-11 18:31:25 [INFO]   ---------------------------------
2020-11-11 18:31:25 [WARN]   Failed to get filetype for:[/tmp/lint/.DS_Store]!
2020-11-11 18:31:25 [WARN]   Failed to get filetype for:[/tmp/lint/.dockerignore]!
2020-11-11 18:31:25 [WARN]   Failed to get filetype for:[/tmp/lint/.gcloudignore]!
2020-11-11 18:31:26 [WARN]   Failed to get filetype for:[/tmp/lint/.gitignore]!
...etc.
```

Kind of makes it difficult to debug when the debug flag makes it work! 😀

Will dig on and see if I can see why this is...

**Comment by bazzadp** [comment]
Curiouser and curious. Just setting the LOG_LEVEL (even to the default VERBOSE) also fixes the problem!

**Comment by bazzadp** [comment]
> Curiouser and curious. Just setting the LOG_LEVEL (even to the default VERBOSE) also fixes the problem!

Sorry this piece is incorrect. I still had `ACTIONS_RUNNER_DEBUG=true` set. When I remove that it reverts back to taking ages while it scans `node_modules` and other folders.

**Comment by GaboFDC** [comment]
Could yo then share an output with only `LOG_LEVEL` set?

**Comment by bazzadp** [comment]
Turns out I was wrong (again!) and `-e ACTIONS_RUNNER_DEBUG=true` didn't actually fix it. I mistook some of the extra logging to mistakenly believe the linter was further along than I thought.

Fix submitted in #1003 and tested with local Docker build and a lot faster.
